api/nextRelease/data/MN/inject.js

```diff
@@ -0,0 +1,15 @@
+const {random, randomItem, pad, range, uppercaseify, include,} = require('../../api');
+
+
+module.exports = (inc, contents) => {
+    const pic = contents.picture;
+    delete contents.picture;
+
+    include(inc, contents, 'phone',  976 + '-' + random(3, 8));
+    include(inc, contents, 'cell',  976 + '-' + random(3, 8));
+    include(inc, contents, 'id', {
+        name: 'RN',
+        value: Math.random().toString(36).replace(/[^a-z]+/g, '').substr(0, 2) + random(3,8);
```

Comment:
Remove semicolon - causing MN to crash.

Reply:
1. Moved MN to api/.nextRelease and removed the unnecessary directory.
2. Converted list files to unix format with the dos2unix.
3. Fixed the bug in "inject.js"